_projects/gitmate_jira.md

```diff
@@ -38,24 +38,30 @@ mentors:
   - fneu
 name: "Implement Jira Support for GitMate"
 requirements:
-  - "The participant should have at least one patch to [IGitt](https://gitlab.com/gitmate/open-source/IGitt/) accepted."
-  - "The participant should have at least one patch to [GitMate](https://gitlab.com/gitmate/open-source/gitmate-2) accepted."
+  - "The participant should have at least one patch to 
+	 [IGitt](https://gitlab.com/gitmate/open-source/IGitt/) accepted."
```

Comment:
align indented text on next line under first letter of above line.

(also applies to next bullet item)